lib/versioning/loose/index.spec.ts

```diff
@@ -40,10 +40,23 @@ describe('versioning/loose/index', () => {
   test.each`
     a             | b              | expected
     ${'2.4.0'}    | ${'2.4'}       | ${true}
```

Comment:
@rarkins What's the case for `2.4` to be higher than `2.4.0`, i.e. not equal? Can we make them equal in order to simplify the code?

Reply:
I think we could/should make these equal

Reply:
This PR is ready for review then